tests/test-app/package.json

```diff
@@ -6,7 +6,7 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
-    "imagekitio-react": "../../",
+    "imagekitio-react": "1.1.1",
```

Comment:
We should remove imagekitio-react dependency from here since we are installing it from tar in CI - https://github.dev/imagekit-developer/imagekit-react/blob/49a2f0acce803de1dd43be37f83a676e802ac6fb/.github/workflows/nodejs.yml#L57-L58

```bash
npm install && npm install ../../imagekitio*.tgz && npm run build
```